src/jdk.charsets/share/classes/sun/nio/cs/ext/IBM942C.java.template

```diff
@@ -66,7 +66,7 @@ public class IBM942C extends Charset implements HistoricallyNamedCharset
         static final char[] c2bIndex;
 
         static {
-            // the mappings need udpate are
+            // the mappings that need updating are
```

Comment:
It probably meant to say "needing" but what you have is okay.

Reply:
That was actually my first approach, but my English intuition failed me in finding the rest of the sentence. Should it be "the mappings needing update are", "the mappings needing an update are" or "the mappings needing updates are"? Neither sounded really good. So I went by a safer route. :)